docs/events/workshops/posts/2024-workshop.md

```diff
@@ -4,4 +4,15 @@ date: 2024-02-13
 
 # G-ADOPT Workshop 2024
 
-The 2024 G-ADOPT workshop will be held away from the ANU, at the NSW south coast. Mark your diaries for 13/11/24-15/11/24. Further information to follow.
\ No newline at end of file
+The 2024 G-ADOPT workshop will be the first held away from Canberra, at the [ANU's Kioloa Coastal Campus](https://www.anu.edu.au/about/campuses-facilities/kioloa-coastal-campus). Mark your diaries for Sunday 10/11/24 - Tuesday 12/11/24. 
```

Comment:
Not sure about `the first held away from Canberra` being useful here. Perhaps say that Kioloa is in NSW, north of Batemans Bay.